Tests/ClientTest.php

```diff
@@ -116,14 +120,11 @@ public function testAuthJson()
 	}
 
 	/**
-	 * Tests the isauth method
-	 *
-	 * @group   JOAuth2
-	 * @return  void
+	 * Tests the isAuth method
```

Comment:
It's not a hard requirement.  Unless that's what the testing group has decided.

You can pass a `--testdox` argument when executing PHPUnit which goes through your test methods and turns your test methods into essentially a description of what's being tested.  It can be used as a way to describe your API.  In this case, the testdox output would print "Is Auth" for the run.

The `@testdox` annotation can be used to further customize the output message.  Say we're testing that the user is authenticated and we named the test `testValidateIsAuthReturnsFalseIfUserIsNotAuthenticated` then the printout would be "Validate Is Auth Returns False If User Is Not Authenticated".  This isn't optimal if the method is named isAuth.  So we could use `@testdox  Validate isAuth() returns false if user is not authenticated` instead to get a more accurate message.